framework/infrastructure/database/mysql/BaseRepositoryMysql.ts

```diff
@@ -50,4 +96,46 @@ export default abstract class BaseRepositoryMysql<T> implements IRepositoryGener
   public rollback() {
     this.DbContext.rollback();
   }
+
+  private amountSearchQueryIncludes(query: IQueryRequest){
+     
+    let filterQ = Array.isArray(query.fields)
+      ? query.fields
+      : query.fields !== undefined
+      ? new Array(query.fields)
+      : [];
+
+    const queryIncludes = Array.isArray(query.includes) ? query.includes : query.includes !== undefined ? new Array(query.includes) : [] ;
+
+    const queryIncludesList = [];
+
+    if ( queryIncludes.length > 0 ) {
+
+      let include = {
+        required:false
+      };
+
+      queryIncludes.forEach(includeQuery => {
+        const model = this.db.sequelize.models[includeQuery];
+        include['model'] = model;
+
+        const whereInclude = filterQ.filter(q => q.indexOf(includeQuery) >= 0);
+
+        let searchableFieldsIncludes = {};
+        whereInclude.forEach(query => {
+          filterQ.splice(filterQ.indexOf(query) , 1);
+          const filter = query.split("=");
+          const field = filter[0].split(".");
+          searchableFieldsIncludes[field[1]] = filter[1];
```

Comment:
Se não tiver o '=' na query, isso vai quebrar...

Reply:
Nessa caso a própria classe que vai montar a query para fazer o filtro, sendo assim sempre vai ter o igual se algum parâmetro for passado.